### PR TITLE
Please Add - Basic Scheme Authentication for ASP.NET Core

### PR DESCRIPTION
**Pull request by mihirdilip**
Added [Mihir.AspNetCore.Authentication.Basic](https://github.com/mihirdilip/Mihir.AspNetCore.Authentication.Basic) - Basic Scheme Authentication for ASP.NET Core to Authentication and Authorization. It is useful for setting up a quick basic authentication for an ASP.NET Core app.

Download directly from https://www.nuget.org/packages/Mihir.AspNetCore.Authentication.Basic
Or by running the below command on your project.
PM> Install-Package Mihir.AspNetCore.Authentication.Basic

Please let me know if you have any queries.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,6 +198,7 @@ Follows best practices and conventions to provide you a SOLID development experi
 * [stormpath-sdk](https://github.com/stormpath/stormpath-sdk-dotnet) - Build [simple, secure web applications](https://github.com/stormpath/stormpath-aspnetcore) with Stormpath and ASP.NET Core. 
 * [stormpath-sdk](https://github.com/stormpath/stormpath-sdk-dotnet) - Build [simple, secure web applications](https://github.com/stormpath/stormpath-aspnetcore) with Stormpath and ASP.NET Core.(Deprecated: It will longer get updated as of March 2017 after joining OKTA) 
 * [stuntman](https://github.com/ritterim/stuntman) - Library for impersonating users during development leveraging ASP.NET Identity.
+* [Mihir.AspNetCore.Authentication.Basic](https://github.com/mihirdilip/Mihir.AspNetCore.Authentication.Basic) - Basic Scheme Authentication for ASP.NET Core
```
Comment (ngohungphuc) [inline README.md]: Please order into alphabet

Comment (mihirdilip) [reply README.md]: I haves changed the order

**Comment by mihirdilip** [comment]
I have changed the order.. thank you

**Comment by mihirdilip** [comment]
Hi @ngohungphuc, am I still missing something?

**Pull request closed without merging**